msgpack/fallback.py

```diff
@@ -212,7 +205,12 @@ class Unpacker(object):
             for o in unpacker:
                 process(o)
     """
-
+    # Some old pythons don't support `self._unpack_from()` `bytearray`. 
+    _unpack_from = staticmethod(
+        struct.unpack_from if sys.version_info >= (2, 7, 6)
+        else lambda f,b,o=0: struct.unpack_from(f, b[:o+2].tobytes(),o)
+    )
```

Comment:
Use global function instead of staticmethod.

Reply:
Sorry it took so many tries.   I have a head cold so it was like typing three lines of code with my toes this morning.